core/README.md

```diff
@@ -0,0 +1,81 @@
+# RPG Toolkit Core
+
+The core module provides fundamental interfaces and types that form the foundation of the RPG toolkit.
+
+## Overview
+
+This module defines the essential building blocks for creating RPG game systems, including:
+
+- **Entity Interface**: The base interface that all game objects must implement
+- **Error Definitions**: Common error types used throughout the toolkit
+
+## Installation
+
+```bash
+go get github.com/KirkDiggler/rpg-toolkit/core
+```
+
+## Usage
+
+### Implementing the Entity Interface
+
+All game objects in the RPG toolkit must implement the `Entity` interface:
+
+```go
+type Entity interface {
+    GetID() string
+    GetType() string
```

Comment:
Should GetType() return a typed constant?

Reply:
the core cannot know about all the types. we will likely have a registration system in place later. For now we are keeping it simple